src/load

```diff
@@ -16,6 +16,8 @@ Copyright (c) 2020 to date, Binare Oy (license@binare.io) All rights reserved.
 """
 
 import argparse
+from turtle import update
```

Comment:
What is this used for? seems like an unnecessary import from a non installed library.